src/picongpu/include/simulation_defines/unitless/ionizerConfig.unitless

```diff
@@ -20,5 +20,6 @@
 
 #pragma once
 
+#include "particles/ionization/byField/ADK/AlgorithmADK.hpp"
```

Comment:
Those are not the same. 
`ionizerConfig.unitless` contains includes of the pure calculation of the models because they are working with physical quantities and might require transformation to unitless ones.   
`particles/ionization/byField/ionizers.hpp` however contains all solvers - the pure informatics part so to say